### ISSUE TITLE
[BUG] Successful/failed chart does not match local time

### ISSUE THREAD

**Issue by TomokoK** [henrywhitaker3/Speedtest-Tracker#495]
**Describe the bug**
The successful/failed tests chart on the web ui uses UTC, but every other chart (e.g. specific tests on the upload/download chart) use the system's local time. 

**To Reproduce**
1. Open the web UI a few minutes before UTC midnight
2. Run tests every minute for a couple minutes before and after UTC midnight
3. Observe the successful/failed chart. Note the date discrepancy between the rest of the chart (assuming you are not GMT).

**Expected behavior**
All charts should be consistent in using either UTC or local time (preferably local time?)

**Context**
 - OS: Docker on RPI 4B
 - Speedtest-Tracker Version: 1.10.0 (latest as of report)

**Comment by bcurran3** [comment]
You do have TZ environment variable set right?

**Comment by TomokoK** [comment]
I have the timezone set to America/New_York. NTP is on and `date` reports the correct date/time.
![1614546990](https://user-images.githubusercontent.com/13804161/109433873-925aee00-79e0-11eb-9f70-ae95f28e90da.png)

**Comment by TomokoK** [comment]
Quick update, my the success/fail chart has completely stopped working for me now. The ping and upload/download charts are still working, however. If someone could point me to the relevant log files I'd be happy to upload.

**Comment by TomokoK** [comment]
I was able to (probably temporarily) fix it by backing up the database, bulk deleting the data, then restoring the database.

**Comment by bcurran3** [comment]
I know we're derailed here, but I had to inject my "me too." I had this problem. All tests stopped 02/15 and wouldn't update even when I manually told it to for almost two weeks. I may or might not have rebooted the server in that time frame, probably not, but I bounced the container two days ago and it started working again.

With that said, I think I've got the problem again as testing stopped again yesterday. :(

I think we need to move this to a new issue. I'll be happy to provide any info to help solve the problem.

![image](https://user-images.githubusercontent.com/14026600/109755135-3d62d780-7b9a-11eb-96cb-85e9305c6d21.png)
